xbmc/video/PlayerController.h

```diff
@@ -46,6 +46,12 @@ class CPlayerController : public ISliderCallback
    */
   virtual void OnSliderChange(void *data, CGUISliderControl *slider);
 
+protected:
+  CPlayerController();
+  CPlayerController(const CPlayerController&);
+  CPlayerController& operator=(CPlayerController const&);
+  virtual ~CPlayerController();
```

Comment:
I've to disagree, if it's declared as a static member or static local within its own class, then the destructor may be private/protected.